spec/factories/supporters.rb

```diff
@@ -9,6 +9,16 @@
         create(:active_card_1, holder: supporter)
       }
     end
+
+    trait :with_primary_address do
+      addresses { [build(:supporter_address)]}
+      primary_address { addresses.first}
+    end
+
+    trait :with_two_addresses do
+      addresses { [build(:supporter_address), build(:other_supporter_address)]}
+      primary_address { addresses.first}
+    end
```

Comment:
They can't have two addresses right now in part 2. It's just not allowed. I guess there's no reason we can't test for it though.